src/index.ts

```diff
@@ -4,7 +4,18 @@ export { default as Accordion } from "./components/Accordion/Accordion";
 export { default as Breadcrumbs } from "./components/Breadcrumbs/Breadcrumbs";
 export { default as Button } from "./components/Button/Button";
 export { ButtonTypes } from "./components/Button/ButtonTypes";
-export { default as Card } from "./components/Card/Card";
+export {
```

Comment:
This is missing `CardImage`, CC @wluisi .

Reply:
@bigfishdesign13 Do you have to time to make this change? I can do it if not, let me know. We just need to export the `CardImage` properly after the development merge on this branch.

Reply:
@nypl-wluisi `CardImage` is now only used internally within `Card`.  Does it still need to be exported in `index.ts`?

Reply:
@bigfishdesign13 @EdwinGuzman Ok, so my issue here with the missing `CardImage` was because of changes to how the component works. I don't think we need to export this child component, as consuming apps can now do this:

```jsx
<Card
  imageComponent={
    <Image
      id={item.image.id}
      alt={item.image.alt}
      uri={item.image.uri}
      useTransformation={true}
      transformations={item.image.transformations}
      transformationLabel={"2_1_960"}
      layout="responsive"
      width={900}
      height={450}
      quality={90}
    />
  }
  imageAspectRatio={CardImageRatios.TwoByOne}
>
```

Reply:
It sounds like we are good here, so I will resolve this comment.

Reply:
I thought it'd be exported even with the new API changes. But yes, we're good now.